en/drafts/originals/transcribe-handwritten-text-with-python-and-azure.md

```diff
@@ -24,132 +24,112 @@ doi: TBC
 
+Regenerating your keys using the button on the **Keys and Endpoint** page is a good way to keep keys secure. When your key changes, just copy and paste it to where you store your key. If you are using this service constantly, logic can be added to your program to use the second key while the first key is being regenerated, which helps avoid any errors.
 
+### Step 4. Creating a Python notebook
 
-### 4. Create a Python notebook
+a) Go to: [https://colab.research.google.com/](https://colab.research.google.com/) (Google Colab is recommended for this lesson, but you can use another Python environment of your choice, such as Anaconda. See the lesson by Quinn Dombrowski, Tassie Gniady, and David Kloster, ["Introduction to Jupyter Notebooks"](/en/lessons/jupyter-notebooks).)
 
-1\. Go to: [https://colab.research.google.com/](https://colab.research.google.com/) (Google Colab is recommended for this lesson, but you can use another Python environment of your choice, such as Anaconda. See the lesson by Quinn Dombrowski, Tassie Gniady, and David Kloster, "Introduction to Jupyter Notebooks.[^1]))
+b) Click _New Notebook_ in the dialog box that opens. Clicking _File_ \| _New notebook_ in the menu will do the same thing.
 
-2\. Click _New Notebook_ in the dialog box that opens. Clicking _File_ \| _New notebook_ in the menu will do the same thing.
+c) When the notebook opens, give it a new title at the top: `Transcribe handwriting and text with Microsoft Azure Cognitive Services.ipynb`.
 
-3\. When the notebook opens, give it a new title at the top: "Transcribe handwriting and text with Microsoft Azure Cognitive Services.ipynb"
-
-4\. The code below will store your Key and Endpoint in an environment variable so that it can be accessed by the program. In Google Colab an empty text box or "cell" to write code in should be visible, if not click the "+ Code" button to add a new code cell. Copy the code below into a cell in your notebook. Edit line 7 to change `https://computer-vision-transcription-jhb.cognitiveservices.azure.com/` to the Endpoint URL you created in the section above. Line numbers are visible in Google Colab and noted in some program comments.
+d) The code below will store your key and endpoint in an environment variable which will make it accessible by the program. In Google Colab, you should be able to see an empty text box or "cell" in which to write code. If not, click the _+ Code_ button to add a new code cell. Copy the code below into a cell in your notebook. Change `https://computer-vision-transcription-jhb.cognitiveservices.azure.com/` to the endpoint URL you created in Step 3.
 
 ```
 import os
 
 print('Enter your secret computer vision key:')
 cv_key = input()
 
-# Change the cv_endpoint below to your endpoint. Line 6
```

Comment:
In dialogue with Alex, I have removed references to line numbers that appear within your code blocks throughout the lesson. It is unclear which lines these refer to, and your explanations are already so detailed that the reader should be able to navigate around the code themselves!